airbyte-integrations/connectors/source-stripe/streams/authorizations.patch.json

```diff
@@ -0,0 +1,21 @@
+{
+    "properties": {
+        "transactions": {
+            "items": {
+                "properties": {
+                    "purchase_details": {
+                        "fuel": null,
```

Comment:
These properties have been wrongly put under `purchase_details` by airbyte, but they should be under `purchase_details.properties`, they are properties of `purchase_details`. You have correctly marked the wrong ones with `null` to remove them, but you need to put them in their correct place, nested under a `properties`, something like:

```
"purchase_details": {
  "properties": {
    "fuel": {
      // type of this value
    },
    "lodging": ...
  }
}
```

Reply:
Instead of removing the files, they should be under properties?